README.md

```diff
@@ -1,49 +1,54 @@
 [![Build Status](https://travis-ci.com/OSGeo/grass.svg?branch=master)](https://travis-ci.com/OSGeo/grass)
 
-GRASS GIS Repository
-====================
+# GRASS GIS Repository
 
-GRASS GIS ([https://grass.osgeo.org/](https://grass.osgeo.org/)) is a Geographic Information System
-used for geospatial data management and analysis, image processing, graphics/map production,
-spatial modeling, and visualization.
+## Description
 
+GRASS GIS ([https://grass.osgeo.org/](https://grass.osgeo.org/)) is
+a Geographic Information System used for geospatial data management and
+analysis, image processing, graphics/map production, spatial modeling, and
+visualization.
 
-How to get write access here?
------------------------------
+## How to get write access here?
 
-In general: you don't really need write access as you can simply open a
-[pull request](https://github.com/OSGeo/grass/pulls) to contribute to GRASS GIS.
+In general: you don't really need write access as you can simply open
+a [pull request](https://github.com/OSGeo/grass/pulls) to contribute to
+GRASS GIS.
 
 Want to become a core developer? See
 [Procedure for gaining Git write access](https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASScorerepository)
 
-How to compile GRASS?
----------------------
+## How to compile GRASS?
 
 > See INSTALL file.
 
 Yes, you should really read INSTALL.
```

Comment:
To make it easier, I would put the link to this `INSTALL` file here and maybe also to the wiki: https://grasswiki.osgeo.org/wiki/Compile_and_Install

Reply:
Good point. I will do it in the next commit. I also think that there should be noted how to open a ticket/why the issues are still not available on git in the README.